tests/cli/basictx.sh

```diff
@@ -23,7 +23,7 @@ test00GetAccount() {
 
   checkAccount $SENDER "0" "9007199254740992"
 
-  ACCT2=$(${CLIENT_EXE} query account $RECV)
+  ACCT2=$(${CLIENT_EXE} query account $RECV 2>/dev/null)
```

Comment:
It is supposed to fail. And I want no output on failure.

Rigel changed so all errors go to stderr.  So unless I want lots of error messages in a successful test, I have to do this if the next line is `assertFalse $?`